zenoh-perf/Cargo.toml

```diff
@@ -32,7 +32,7 @@ maintenance = { status = "actively-developed" }
 
 
 [dependencies]
-async-std = { version = "=1.8.0", features = ["unstable"] }
+async-std = { version = "=1.9.0", features = ["unstable"] }
```

Comment:
features = ["unstable"] was needed until async-std 1.8.0 to support the async channels. 
Since the channels have been stabilised in async-std 1.9.0, features = ["unstable"] should be hence removed.